### PR TITLE
React on rails

### PR DESCRIPTION
**Pull request by dennisyang101**
Environment check ok, merge with main

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: Procfile.dev-hmr [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Procfile for development using HMR
```
Comment (SAALICE) [inline Procfile.dev-hmr:1]: 這份跟上面好像一樣喔～

**Review by dream184 — APPROVED** [review]
ok

**Review by ghost — APPROVED** [review]
okok

**Review by SAALICE — APPROVED** [review]

**Review by waliwang — APPROVED** [review]
good

**Review by jeter1208 — APPROVED** [review]

**Pull request merged**